:art: Restructuring of studentgroups

alvarhonsi opened the pull request.

bakseter commented:
Må vel ha ei litta migrasjon av data (https://www.sanity.io/docs/migrating-data) også først

boaanes commented:
Vi gjør det manuelt, jeg fikser review @bakseter

bakseter commented:
![](https://media3.giphy.com/media/gf58YQbHE6lX6vItke/giphy.gif)

boaanes reviewed (approved):
LGTM

alvarhonsi committed:
:art: Restructured the contentmodel of student groups

Embedded student group role relations directly into the student group document, insted of having it as a seperate document to be referenced
